content/rs/references/rest-api/_index.md

```diff
@@ -20,75 +20,105 @@ For more info, see:
 
-Some responses may have an empty body but indicate the response with standard [HTTP codes](https://www.w3.org/Protocols/rfc2616/rfc2616-sec10.html).
+Example request using [cURL](https://curl.se/):
 
-Both requests and responses may include zero or more objects.
+``` bash
+curl -u "demo@redislabs.com:password" \
+    https://localhost:9443/v1/bdbs
+```
 
-If the request is for a single entity, the response returns a single JSON object or none. If the request is for a list of entities, the response returns a JSON array with zero or more elements.
+For more examples, see the [Redis Enterprise Software REST API quick start]({{<relref "/rs/references/rest-api/quick-start" >}})
 
-If you omit certain JSON object fields from a request, they may be assigned default values, which often indicate that these fields are not in use.
+### Permissions
 
-### Request headers
+By default, the admin user is authorized for access to all endpoints. Use [role-based access controls]({{< relref "/rs/security/admin-console-security/user-security.md" >}}) and [role permissions]({{<relref "/rs/references/rest-api/permissions/_index.md" >}}) to manage access.
 
-Redis Enterprise REST API requests support the following HTTP headers:
+If a user attempts to access an endpoint that is not allowed in their role, the request will fail with a [`403 Forbidden`](https://www.rfc-editor.org/rfc/rfc9110.html#name-403-forbidden) status code. For more details on which user roles can access certain endpoints, see [Permissions]({{<relref "/rs/references/rest-api/permissions">}}).
 
-| Header | Supported/Required Values |
-|--------|---------------------------|
-| Accept | `application/json` |
-| Content-Length | Length (in bytes) of request message |
-| Content-Type | `application/json` |
+### Certificates
 
-### Response headers
+The Redis Enterprise Software REST API uses [Self-signed certificates]({{<relref "/rs/security/certificates">}}) to ensure the product is secure. When you use the default self-signed certificates, the HTTPS requests will fail with `SSL certificate problem: self signed certificate` unless you turn off SSL certificate verification. The examples in this tutorial turn off SSL certificate verification.
 
-Redis Enterprise REST API responses support the following HTTP headers:
+## Ports
 
-| Header | Supported/Required Values |
-|--------|---------------------------|
-| Content-Type | `application/json` |
-| Content-Length | Length (in bytes) of response message |
+All calls must be made over SSL to port 9443. For the API to work, port 9443 must be exposed to incoming traffic or mapped to a different port.
+
+If you are using a [Redis Enterprise Software Docker image]({{<relref "/rs/installing-upgrading/get-started-docker">}}), run the following command to start the Docker image with port 9443 exposed:
+
+```sh
+docker run -p 9443:9443 redislabs/redis
+```
+
+## JSON Requests
```

Comment:
Also, maybe you should go ahead and move this down below the request/response headers section and before the response types/error codes.
```suggestion
## JSON requests and responses
```